tests/cli/test_rasa_data.py

```diff
@@ -156,7 +157,7 @@ def test_validate_files_action_not_found_invalid_domain(
     file_name = tmp_path / f"{file_type}.yml"
     file_name.write_text(
         f"""
-        version: "3.0"
+        version: "{LATEST_TRAINING_DATA_FORMAT_VERSION}"
```

Comment:
Could we actually use the constant directly without quotation marks around it? That would also prevent the failing unit tests.

Reply:
I think it does want some form of quotes based on this error - but singles, not doubles because of the way we've written the test data. I also noticed that during a migration it ends up writing `'"3.0'"` which then isn't read correctly, but I figured that's a separate issue. I'll see which combo of quotes makes it happy
```        
domain = Domain.from_yaml(test_yaml)
>       assert domain.as_yaml(clean_before_dump=True) == test_yaml
E       assert "version: '3....: from_text\n" == 'version: "3....: from_text\n'
E         Skipping 983 identical trailing characters in diff, use -v to show
E         - version: "3.0"
E         ?          ^   ^
E         + version: '3.0'
```

Reply:
Well, actually maybe I should fix that too because it's going to give an error regardless:
```
>           assert migrated_training_data_version == LATEST_TRAINING_DATA_FORMAT_VERSION
E           assert '"3.0"' == '3.0'
E             - 3.0
E             + "3.0"
```

Reply:
Isn't it because `LATEST_TRAINING_DATA_FORMAT_VERSION` is already assigned a string `"3.0"`? So when added to yaml, you don't need more quotes?

Reply:
I think you're right, lemme check and try the tests again